.github/workflows/mumuk-ci-cd.yml

```
@@ -69,25 +73,11 @@ jobs:
           username: ${{ secrets.SERVER_USERNAME }}
           key: ${{ secrets.SERVER_SSH_KEY }}
           script: |
-            cd /home/ubuntu/BE
+            docker login -u "${{ secrets.DOCKERHUB_USERNAME }}" -p "${{ secrets.DOCKERHUB_PASSWORD }}"
+            docker pull ${{ secrets.DOCKERHUB_USERNAME }}/mumuk-backend:latest            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)